frontend/src/store/account.js

```diff
@@ -148,6 +148,13 @@ const actions = {
             const user = await userApi.getUser()
             state.commit('login', user)
 
+            if (window._ffhstc) {
```

Comment:
We have a service that handles this for posthog and sentry, this identify command can live there too:

https://github.com/FlowFuse/flowfuse/blob/aadd6091dd905fb0ee1626a39e75874c5c14c69c/frontend/src/services/product.js#L9-L17